aspnetcore/mvc/advanced/app-parts/3.0sample1/WebAppParts/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace WebAppParts
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder
+                     .UseStartup<Startup>();
+                    // .UseStartup<Startup2>();
```

Comment:
?

Reply:
I have two different startups for showing different features. They each need to be a snippet. This is a typical pattern we follow in the docs.